src/DeviceInput/deviceInputSystem.ts

```diff
@@ -411,6 +456,36 @@ export class DeviceInputSystem implements IDisposable {
 
         });
 
+        this._pointerBlurEvent = ((evt) => {
+            // Handle mouse buttons
+            if (this.isDeviceAvailable(DeviceType.Mouse)) {
+                const pointer = this._inputs[DeviceType.Mouse][0];
+                for (let i = 0; i < DeviceInputSystem._MAX_POINTER_INPUTS; i++) {
+                    if (pointer[i + 2] === 1) {
+                        pointer[i + 2] = 0;
+
+                        if (this.onInputChanged) {
+                            this.onInputChanged(DeviceType.Mouse, 0, i + 2, 1, pointer[i + 2]);
+                        }

```

Comment:
I don't understand why we need to add fake events.

Reply:
Talked offline